src/components/ConfirmationModal.tsx

```diff
@@ -58,8 +70,26 @@ const ConfirmationModal = (props: ConfirmationModalProps): JSX.Element => {
             <CloseIcon width={wp('6%')} height={wp('6%')} />
           </Icon>
           <Message>{text}</Message>
-          <Button label={cancelButtonText} onPress={closeModal} buttonTheme={BUTTONS_THEME.contained} />
-          <Button label={confirmationButtonText} onPress={confirmationAction} buttonTheme={BUTTONS_THEME.outlined} />
+          {children}
+          {lockingModal ? (
+            <>
+              <Button
+                label={confirmationButtonText}
+                onPress={confirmationAction}
+                buttonTheme={BUTTONS_THEME.contained}
+              />
+              <Button label={cancelButtonText} onPress={closeModal} buttonTheme={BUTTONS_THEME.outlined} />
+            </>
+          ) : (
+            <>
+              <Button label={cancelButtonText} onPress={closeModal} buttonTheme={BUTTONS_THEME.contained} />
+              <Button
+                label={confirmationButtonText}
+                onPress={confirmationAction}
+                buttonTheme={BUTTONS_THEME.outlined}
+              />
```

Comment:
Fine for now but in the future we should probably generalize this. The only difference right now is the order of the buttons, right?

Reply:
@klinzo should I /can you open a ticket for it?

Reply:
Sure: https://issues.tuerantuer.org/browse/LUN-312